src/stepper/Stepper.unit.tsx

```diff
@@ -37,7 +38,7 @@ it('Should have the default text & attributes', () => {
     />,
   )
 
-  expect(stepper.find('.kirk-stepper').prop('aria-label')).toEqual(
+  expect(stepper.find('.kirk-stepper-content').prop('aria-label')).toEqual(
```

Comment:
please, target components instead of classes

Reply:
There was no decision made on this topic yet, Simon has started a documents on how we write tests, but this still in progress. So for now targeting classes is fine.

Reply:
No component here to target, and I won't rewrite all of it right now ;)